model/common/src/icon4py/model/common/grid/horizontal.py

```diff
@@ -420,6 +420,11 @@ def _domain(marker: Zone) -> Domain:
     return _domain
 
 
+vertex_domain = domain(dims.VertexDim)
+edge_domain = domain(dims.EdgeDim)
+cell_domain = domain(dims.CellDim)
+
+
```

Comment:
Are these used somewhere?

Reply:
I import them `geometry.py`. We have been redefining them all over the place, and I got  a bit tired of that.